shell/platform/darwin/ios/BUILD.gn

```diff
@@ -153,6 +143,142 @@ shared_library("create_flutter_framework_dylib") {
+    "../../../../../third_party/ocmock/Source/OCMock/OCMIndirectReturnValueProvider.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMIndirectReturnValueProvider.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMInvocationExpectation.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMInvocationExpectation.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMInvocationMatcher.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMInvocationMatcher.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMInvocationStub.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMInvocationStub.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMLocation.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMLocation.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMMacroState.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMMacroState.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMNotificationPoster.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMNotificationPoster.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMObserverRecorder.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMObserverRecorder.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMPassByRefSetter.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMPassByRefSetter.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMRealObjectForwarder.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMRealObjectForwarder.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMRecorder.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMRecorder.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMReturnValueProvider.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMReturnValueProvider.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMStubRecorder.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMStubRecorder.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMVerifier.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMVerifier.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMock.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMockObject.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCMockObject.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCObserverMockObject.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCObserverMockObject.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCPartialMockObject.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCPartialMockObject.m",
+    "../../../../../third_party/ocmock/Source/OCMock/OCProtocolMockObject.h",
+    "../../../../../third_party/ocmock/Source/OCMock/OCProtocolMockObject.m",
+  ]
+  include_dirs = [ "../../../../../third_party/ocmock/Source/" ]
+}
+
+ios_test_flutter_path = rebase_path("$root_out_dir/libios_test_flutter.dylib")
```

Comment:
perhaps still mention that it's for simulators in the name

Reply:
added comment